### ISSUE TITLE
ComboBox errorMessage is not read by screen reader

### ISSUE THREAD

**Issue by mirajp** [OfficeDev/office-ui-fabric-react#10693]
### Environment Information

- **Package version(s)**: version 7 (latest at time of writing = 7.42.0)
- **Browser and OS versions**: Chrome 77.0.3865.90 and Edge 44.18362.329.0 (on Windows 10)

### Describe the issue:
Neither Narrator with Edge nor NVDA with Chrome reads/announces the error message of a ComboBox component when the error is set/made visible.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/mirajp/pen/jONgJag

#### Actual behavior:
Error message shows up but is not read aloud.

#### Expected behavior:
Error message to show up and and be read aloud as an alert.

### Documentation describing expected behavior
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html ?

(I noticed that the rendered element doesn't have any special roles/aria-live attributes).

**Comment by khmakoto** [comment]
Hi @mirajp, thanks for filing this bug! I can confirm that this is reproing form me in the codepen you provided (thanks for that!). I'll work on a fix as soon as I can.
